pkg/detector/detector.go

```diff
@@ -699,27 +718,9 @@ func (d *ResourceDetector) GetUnstructuredObject(objectKey keys.ClusterWideKey)
 	return unstructuredObj, nil
 }
 
-func (d *ResourceDetector) getPropagationPolicyID(policy *policyv1alpha1.PropagationPolicy) (string, error) {
```

Comment:
The operation of obtaining the PermanentID involves version upgrade. We can delete this logic after we add PermanentID through `karmada-webhook`.

Reply:
Okay, I'll restore it. It will be removed in the PR which handles the deletion of labels.